packages/sdk-react/src/core/context/MoniteProvider.tsx

```diff
@@ -232,9 +231,7 @@ export const MoniteProvider = ({
         key={moniteInstanceKey}
         sentryHub={sentryHub}
       >
-        <MoniteThemeContext.Provider
-          value={theme ?? defaultMoniteLightThemeOptions}
-        >
+        <MoniteThemeContext.Provider value={theme}>
```

Comment:
If our user migrates to a new version and doesn't pass any `theme` the theme will be set to a Material, won't it?
If it's true isn't it a major release?

Reply:
yes, you are right.
I discussed it with Nikita. We agreed that it is not a breaking change, everything will work, just the UI will look a little bit different.
Probably all our current customers use their own theme.

Reply:
Hm, okay. I hope so